### PR TITLE
retroarch: 1.10.3 -> 1.11.0;  libretro: unstable-2022-04-21 -> unstable-2022-10-01

### PR DESCRIPTION
**Pull request by thiagokokada**
###### Description of changes

New release: https://www.libretro.com/index.php/retroarch-1-11-0-release/

This PR also introduces the new `libretro` team.

Also as it is common for each update, one more round of clean-ups.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/team-list.nix
Comment (thiagokokada) [inline maintainers/team-list.nix:369]: @aanderse I add you to this team since you're always helping with reviews. If you don't want it just ask me and I can remove your name.

Comment (thiagokokada) [reply maintainers/team-list.nix:369]: Pinging @edwtjo and @MP2E too, I added you both to this team since you're already maintainers of both the RetroArch and libretro packages. If you don't want to be here just give me a call.

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 193939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>91 packages built:</summary>
  <ul>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>libretro.atari800</li>
    <li>libretro.beetle-gba</li>
    <li>libretro.beetle-lynx</li>
    <li>libretro.beetle-ngp</li>
    <li>libretro.beetle-pce-fast</li>
    <li>libretro.beetle-pcfx</li>
    <li>libretro.beetle-psx</li>
    <li>libretro.beetle-psx-hw</li>
    <li>libretro.beetle-saturn</li>
    <li>libretro.beetle-snes</li>
    <li>libretro.beetle-supergrafx</li>
    <li>libretro.beetle-vb</li>
    <li>libretro.beetle-wswan</li>
    <li>libretro.blastem</li>
    <li>libretro.bluemsx</li>
    <li>libretro.bsnes</li>
    <li>libretro.bsnes-hd</li>
    <li>libretro.bsnes-mercury</li>
    <li>libretro.bsnes-mercury-balanced</li>
    <li>libretro.bsnes-mercury-performance</li>
    <li>libretro.citra</li>
    <li>libretro.desmume</li>
    <li>libretro.desmume2015</li>
    <li>libretro.dolphin</li>
    <li>libretro.dosbox</li>
    <li>libretro.eightyone</li>
    <li>libretro.fbalpha2012</li>
    <li>libretro.fbneo</li>
    <li>libretro.fceumm</li>
    <li>libretro.flycast</li>
    <li>libretro.fmsx</li>
    <li>libretro.freeintv</li>
    <li>libretro.gambatte</li>
    <li>libretro.genesis-plus-gx</li>
    <li>libretro.gpsp</li>
    <li>libretro.gw</li>
    <li>libretro.handy</li>
    <li>libretro.hatari</li>
    <li>libretro.mame</li>
    <li>libretro.mame2000</li>
    <li>libretro.mame2003</li>
    <li>libretro.mame2003-plus</li>
    <li>libretro.mame2010</li>
    <li>libretro.mame2015</li>
    <li>libretro.mame2016</li>
    <li>libretro.melonds</li>
    <li>libretro.mesen</li>
    <li>libretro.mesen-s</li>
    <li>libretro.meteor</li>
    <li>libretro.mgba</li>
    <li>libretro.mupen64plus</li>
    <li>libretro.neocd</li>
    <li>libretro.nestopia</li>
    <li>libretro.np2kai</li>
    <li>libretro.nxengine</li>
    <li>libretro.o2em</li>
    <li>libretro.opera</li>
    <li>libretro.parallel-n64</li>
    <li>libretro.pcsx-rearmed</li>
    <li>libretro.pcsx2</li>
    <li>libretro.picodrive</li>
    <li>libretro.play</li>
    <li>libretro.ppsspp</li>
    <li>libretro.prboom</li>
    <li>libretro.prosystem</li>
    <li>libretro.quicknes</li>
    <li>libretro.sameboy</li>
    <li>libretro.scummvm</li>
    <li>libretro.smsplus-gx</li>
    <li>libretro.snes9x</li>
    <li>libretro.snes9x2002</li>
    <li>libretro.snes9x2005</li>
    <li>libretro.snes9x2005-plus</li>
    <li>libretro.snes9x2010</li>
    <li>libretro.stella</li>
    <li>libretro.stella2014</li>
    <li>libretro.swanstation</li>
    <li>libretro.tgbdual</li>
    <li>libretro.thepowdertoy</li>
    <li>libretro.tic80</li>
    <li>libretro.vba-m</li>
    <li>libretro.vba-next</li>
    <li>libretro.vecx</li>
    <li>libretro.virtualjaguar</li>
    <li>libretro.yabause</li>
    <li>retroarch</li>
    <li>retroarchBare</li>
    <li>retroarchFull</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
Tested:
- Beetle GBA
- Beetle PSX (normal/HW)
- blastem
- bsnes
- bsnes-hd
- flycast
- gpSP
- Meteor
- mGBA
- pcsx-rearmed
- picodrive
- Snes9x
- SwanStation
- VBA Next
- VBA-M

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by thiagokokada** [comment]
Result of `nixpkgs-review pr 193939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>91 packages built:</summary>
  <ul>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>libretro.atari800</li>
    <li>libretro.beetle-gba</li>
    <li>libretro.beetle-lynx</li>
    <li>libretro.beetle-ngp</li>
    <li>libretro.beetle-pce-fast</li>
    <li>libretro.beetle-pcfx</li>
    <li>libretro.beetle-psx</li>
    <li>libretro.beetle-psx-hw</li>
    <li>libretro.beetle-saturn</li>
    <li>libretro.beetle-snes</li>
    <li>libretro.beetle-supergrafx</li>
    <li>libretro.beetle-vb</li>
    <li>libretro.beetle-wswan</li>
    <li>libretro.blastem</li>
    <li>libretro.bluemsx</li>
    <li>libretro.bsnes</li>
    <li>libretro.bsnes-hd</li>
    <li>libretro.bsnes-mercury</li>
    <li>libretro.bsnes-mercury-balanced</li>
    <li>libretro.bsnes-mercury-performance</li>
    <li>libretro.citra</li>
    <li>libretro.desmume</li>
    <li>libretro.desmume2015</li>
    <li>libretro.dolphin</li>
    <li>libretro.dosbox</li>
    <li>libretro.eightyone</li>
    <li>libretro.fbalpha2012</li>
    <li>libretro.fbneo</li>
    <li>libretro.fceumm</li>
    <li>libretro.flycast</li>
    <li>libretro.fmsx</li>
    <li>libretro.freeintv</li>
    <li>libretro.gambatte</li>
    <li>libretro.genesis-plus-gx</li>
    <li>libretro.gpsp</li>
    <li>libretro.gw</li>
    <li>libretro.handy</li>
    <li>libretro.hatari</li>
    <li>libretro.mame</li>
    <li>libretro.mame2000</li>
    <li>libretro.mame2003</li>
    <li>libretro.mame2003-plus</li>
    <li>libretro.mame2010</li>
    <li>libretro.mame2015</li>
    <li>libretro.mame2016</li>
    <li>libretro.melonds</li>
    <li>libretro.mesen</li>
    <li>libretro.mesen-s</li>
    <li>libretro.meteor</li>
    <li>libretro.mgba</li>
    <li>libretro.mupen64plus</li>
    <li>libretro.neocd</li>
    <li>libretro.nestopia</li>
    <li>libretro.np2kai</li>
    <li>libretro.nxengine</li>
    <li>libretro.o2em</li>
    <li>libretro.opera</li>
    <li>libretro.parallel-n64</li>
    <li>libretro.pcsx-rearmed</li>
    <li>libretro.pcsx2</li>
    <li>libretro.picodrive</li>
    <li>libretro.play</li>
    <li>libretro.ppsspp</li>
    <li>libretro.prboom</li>
    <li>libretro.prosystem</li>
    <li>libretro.quicknes</li>
    <li>libretro.sameboy</li>
    <li>libretro.scummvm</li>
    <li>libretro.smsplus-gx</li>
    <li>libretro.snes9x</li>
    <li>libretro.snes9x2002</li>
    <li>libretro.snes9x2005</li>
    <li>libretro.snes9x2005-plus</li>
    <li>libretro.snes9x2010</li>
    <li>libretro.stella</li>
    <li>libretro.stella2014</li>
    <li>libretro.swanstation</li>
    <li>libretro.tgbdual</li>
    <li>libretro.thepowdertoy</li>
    <li>libretro.tic80</li>
    <li>libretro.vba-m</li>
    <li>libretro.vba-next</li>
    <li>libretro.vecx</li>
    <li>libretro.virtualjaguar</li>
    <li>libretro.yabause</li>
    <li>retroarch</li>
    <li>retroarchBare</li>
    <li>retroarchFull</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by AtilaSaraiva** [comment]
I don't feel qualified to make comments about the code, and I can't actually test the package, but I can confirm it builds!

Result of `nixpkgs-review pr 193939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>91 packages built:</summary>
  <ul>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>libretro.atari800</li>
    <li>libretro.beetle-gba</li>
    <li>libretro.beetle-lynx</li>
    <li>libretro.beetle-ngp</li>
    <li>libretro.beetle-pce-fast</li>
    <li>libretro.beetle-pcfx</li>
    <li>libretro.beetle-psx</li>
    <li>libretro.beetle-psx-hw</li>
    <li>libretro.beetle-saturn</li>
    <li>libretro.beetle-snes</li>
    <li>libretro.beetle-supergrafx</li>
    <li>libretro.beetle-vb</li>
    <li>libretro.beetle-wswan</li>
    <li>libretro.blastem</li>
    <li>libretro.bluemsx</li>
    <li>libretro.bsnes</li>
    <li>libretro.bsnes-hd</li>
    <li>libretro.bsnes-mercury</li>
    <li>libretro.bsnes-mercury-balanced</li>
    <li>libretro.bsnes-mercury-performance</li>
    <li>libretro.citra</li>
    <li>libretro.desmume</li>
    <li>libretro.desmume2015</li>
    <li>libretro.dolphin</li>
    <li>libretro.dosbox</li>
    <li>libretro.eightyone</li>
    <li>libretro.fbalpha2012</li>
    <li>libretro.fbneo</li>
    <li>libretro.fceumm</li>
    <li>libretro.flycast</li>
    <li>libretro.fmsx</li>
    <li>libretro.freeintv</li>
    <li>libretro.gambatte</li>
    <li>libretro.genesis-plus-gx</li>
    <li>libretro.gpsp</li>
    <li>libretro.gw</li>
    <li>libretro.handy</li>
    <li>libretro.hatari</li>
    <li>libretro.mame</li>
    <li>libretro.mame2000</li>
    <li>libretro.mame2003</li>
    <li>libretro.mame2003-plus</li>
    <li>libretro.mame2010</li>
    <li>libretro.mame2015</li>
    <li>libretro.mame2016</li>
    <li>libretro.melonds</li>
    <li>libretro.mesen</li>
    <li>libretro.mesen-s</li>
    <li>libretro.meteor</li>
    <li>libretro.mgba</li>
    <li>libretro.mupen64plus</li>
    <li>libretro.neocd</li>
    <li>libretro.nestopia</li>
    <li>libretro.np2kai</li>
    <li>libretro.nxengine</li>
    <li>libretro.o2em</li>
    <li>libretro.opera</li>
    <li>libretro.parallel-n64</li>
    <li>libretro.pcsx-rearmed</li>
    <li>libretro.pcsx2</li>
    <li>libretro.picodrive</li>
    <li>libretro.play</li>
    <li>libretro.ppsspp</li>
    <li>libretro.prboom</li>
    <li>libretro.prosystem</li>
    <li>libretro.quicknes</li>
    <li>libretro.sameboy</li>
    <li>libretro.scummvm</li>
    <li>libretro.smsplus-gx</li>
    <li>libretro.snes9x</li>
    <li>libretro.snes9x2002</li>
    <li>libretro.snes9x2005</li>
    <li>libretro.snes9x2005-plus</li>
    <li>libretro.snes9x2010</li>
    <li>libretro.stella</li>
    <li>libretro.stella2014</li>
    <li>libretro.swanstation</li>
    <li>libretro.tgbdual</li>
    <li>libretro.thepowdertoy</li>
    <li>libretro.tic80</li>
    <li>libretro.vba-m</li>
    <li>libretro.vba-next</li>
    <li>libretro.vecx</li>
    <li>libretro.virtualjaguar</li>
    <li>libretro.yabause</li>
    <li>retroarch</li>
    <li>retroarchBare</li>
    <li>retroarchFull</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
CC @aanderse for review.

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by AtilaSaraiva** [comment]
Result of `nixpkgs-review pr 193939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>libretro.mame2010</li>
    <li>libretro.mame2016</li>
    <li>retroarchFull</li>
  </ul>
</details>
<details>
  <summary>88 packages built:</summary>
  <ul>
    <li>kodiPackages.libretro-genplus</li>
    <li>kodiPackages.libretro-mgba</li>
    <li>kodiPackages.libretro-snes9x</li>
    <li>libretro.atari800</li>
    <li>libretro.beetle-gba</li>
    <li>libretro.beetle-lynx</li>
    <li>libretro.beetle-ngp</li>
    <li>libretro.beetle-pce-fast</li>
    <li>libretro.beetle-pcfx</li>
    <li>libretro.beetle-psx</li>
    <li>libretro.beetle-psx-hw</li>
    <li>libretro.beetle-saturn</li>
    <li>libretro.beetle-snes</li>
    <li>libretro.beetle-supergrafx</li>
    <li>libretro.beetle-vb</li>
    <li>libretro.beetle-wswan</li>
    <li>libretro.blastem</li>
    <li>libretro.bluemsx</li>
    <li>libretro.bsnes</li>
    <li>libretro.bsnes-hd</li>
    <li>libretro.bsnes-mercury</li>
    <li>libretro.bsnes-mercury-balanced</li>
    <li>libretro.bsnes-mercury-performance</li>
    <li>libretro.citra</li>
    <li>libretro.desmume</li>
    <li>libretro.desmume2015</li>
    <li>libretro.dolphin</li>
    <li>libretro.dosbox</li>
    <li>libretro.eightyone</li>
    <li>libretro.fbalpha2012</li>
    <li>libretro.fbneo</li>
    <li>libretro.fceumm</li>
    <li>libretro.flycast</li>
    <li>libretro.fmsx</li>
    <li>libretro.freeintv</li>
    <li>libretro.gambatte</li>
    <li>libretro.genesis-plus-gx</li>
    <li>libretro.gpsp</li>
    <li>libretro.gw</li>
    <li>libretro.handy</li>
    <li>libretro.hatari</li>
    <li>libretro.mame</li>
    <li>libretro.mame2000</li>
    <li>libretro.mame2003</li>
    <li>libretro.mame2003-plus</li>
    <li>libretro.mame2015</li>
    <li>libretro.melonds</li>
    <li>libretro.mesen</li>
    <li>libretro.mesen-s</li>
    <li>libretro.meteor</li>
    <li>libretro.mgba</li>
    <li>libretro.mupen64plus</li>
    <li>libretro.neocd</li>
    <li>libretro.nestopia</li>
    <li>libretro.np2kai</li>
    <li>libretro.nxengine</li>
    <li>libretro.o2em</li>
    <li>libretro.opera</li>
    <li>libretro.parallel-n64</li>
    <li>libretro.pcsx-rearmed</li>
    <li>libretro.pcsx2</li>
    <li>libretro.picodrive</li>
    <li>libretro.play</li>
    <li>libretro.ppsspp</li>
    <li>libretro.prboom</li>
    <li>libretro.prosystem</li>
    <li>libretro.quicknes</li>
    <li>libretro.sameboy</li>
    <li>libretro.scummvm</li>
    <li>libretro.smsplus-gx</li>
    <li>libretro.snes9x</li>
    <li>libretro.snes9x2002</li>
    <li>libretro.snes9x2005</li>
    <li>libretro.snes9x2005-plus</li>
    <li>libretro.snes9x2010</li>
    <li>libretro.stella</li>
    <li>libretro.stella2014</li>
    <li>libretro.swanstation</li>
    <li>libretro.tgbdual</li>
    <li>libretro.thepowdertoy</li>
    <li>libretro.tic80</li>
    <li>libretro.vba-m</li>
    <li>libretro.vba-next</li>
    <li>libretro.vecx</li>
    <li>libretro.virtualjaguar</li>
    <li>libretro.yabause</li>
    <li>retroarch</li>
    <li>retroarchBare</li>
  </ul>
</details>

**Comment by thiagokokada** [comment]
@ofborg build retroarchFull

**Comment by thiagokokada** [comment]
> Result of `nixpkgs-review pr 193939` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
> 3 packages failed to build:
> 
>     * libretro.mame2010
> 
>     * libretro.mame2016
> 
>     * retroarchFull
> 
> 
> 88 packages built:

This doesn't look good, I will disable parallel building for mame2010 and mame2016 them.

**Comment by thiagokokada** [comment]
CC @aanderse for review (sorry for pinging you twice, but now it should be ready).

**Review by aanderse — APPROVED** [review]
Changes LGTM, though have not tested. As always, thanks @thiagokokada!

**Comment by vcunat** [comment]
FYI this PR apparently broke previously working build on aarch64-darwin: https://hydra.nixos.org/job/nixpkgs/nixpkgs-unstable-aarch64-darwin/retroarchBare.aarch64-darwin

**Comment by thiagokokada** [comment]
> FYI this PR apparently broke previously working build on aarch64-darwin: https://hydra.nixos.org/job/nixpkgs/nixpkgs-unstable-aarch64-darwin/retroarchBare.aarch64-darwin

MacOS builds are not working correctly anyway for a bunch of other issues. Maybe I should just mark it as broken.

**Comment by aanderse** [comment]
> Maybe I should just mark it as broken.

Definitely. After that we can ping the darwin team and leave it up to them to fix if they want to.

**Comment by thiagokokada** [comment]
> > Maybe I should just mark it as broken.
> 
> Definitely. After that we can ping the darwin team and leave it up to them to fix if they want to.

Done: https://github.com/NixOS/nixpkgs/pull/195235

**Pull request merged**
